dradis/dradis-ce
# Api whodunnit

Addresses #376

### Spec
Currently, when an item is updated through the API, the "by" attribute of the revision is not updated with the user email.

**Proposed solution**
Add the same method for adding the whodunnit attribute in paper trail.

### How to test

1. Using the API, add an issue
2. Using the API, update the issue
3. Visit the issue's revision history page
4. Confirm that the revision has a value for the "by" attribute.

### Check List
